pkg/operator/ceph/object/user/controller.go

```diff
@@ -503,3 +510,19 @@ func updateStatus(client client.Client, name types.NamespacedName, status string
 	}
 	logger.Debugf("object store user %q status updated to %q", name, status)
 }
+
+//Update the ceph user if needed
+func (r *ReconcileObjectStoreUser) UpdateCephUser(u *cephv1.CephObjectStoreUser) error {
```

Comment:
Instead of creating a new method, it might be preferable inline after the check for `ErrorCodeFileExists`